### ISSUE TITLE
Allow project_root_path to be read from settings.py

### ISSUE THREAD

**Issue by phillipuniverse** [3YOURMIND/django-migration-linter#203]
In the latest change in #199 that fixes #198, we can now get all settings from Django's settings, _except_ for `project_root_path` and `verbosity`. This is because of how early it's read in the process:

https://github.com/3YOURMIND/django-migration-linter/blob/8e56562771df63094349fb667bdb06a872462499/django_migration_linter/management/commands/lintmigrations.py#L141-L146

In my case, my project layout looks like this:

```
.
├── Dockerfile
├── manage.py
├── myapp
│   ├── configuration
│   │   ├── __init__.py
│   │   ├── settings.py
```

And in `manage.py` I default the `DJANGO_SETTINGS_MODULE`:

```python
...
if __name__ == "__main__":
    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "myapp.configuration.settings")
    try:
        from django.core.management import execute_from_command_line
    ...
```

In this case, there is a loose relationship between my project root path and where `settings.py` is (it's actually 2 level up). I think the `lintmigrations` management command assumes that they are the same.

The manifestation of this is that the migration linter comes back with 0 migrations to check, since it's effectively starting at `myapp/configuration`:

```
*** Summary ***
Valid migrations: 0/0
Erroneous migrations: 0/0
Migrations with warnings: 0/0
Ignored migrations: 0/0
```

**Comment by phillipuniverse** [comment]
As a workaround for now, I have this overridden in my project by:

1. Adding the `lintmigrations.py` from #204 to a `management/commands/lintmigrations.py` file in my project
2. Ensuring that my custom application appears in `INSTALLED_APPS` _before_ the `django_migration_linter` entry

With that override, this works great for me with the following in my `settings.py`:

```python
root = environ.Path(__file__) - 2  # two folders back (/a/b/ - 2 = /)

BASE_DIR = root()

MIGRATION_LINTER_OPTIONS = {
    "project_root_path": BASE_DIR
}
```

**Comment by David-Wobrock** [comment]
Thanks a lot for the issue and the PR, checking it out :)
